File: Java/pagamento.java
import java.util.Locale;
import java.util.Scanner;

public class pagamento {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        Locale.setDefault(Locale.US);

        System.out.print("Nome: ");
        String nome = sc.nextLine();
        System.out.print("Valor por hora: ");
        double valor = sc.nextDouble();
        System.out.print("Horas trabalhadas: ");
        int horas = sc.nextInt();

        double pagamento = valor * horas;
        System.out.printf("O pagamento para %s deve ser %.2f", nome, pagamento);

        sc.close();

    }
}

File: C/pagamento.c
#include <stdio.h>

char nome[50];
double valor, horas, total;

int main() {
    printf("Nome: ");
    scanf("%[^\n]", &nome);
    printf("Valor por hora: ");
    scanf("%lf", &valor);
    printf("Horas trabalhadas: ");
    scanf("%lf", &horas);

    total = valor * horas;
    printf("O pagamento para %s deve ser %.2lf", nome, total);
}
